src/lib/utilities.ts

```diff
@@ -62,145 +62,62 @@ function toPascalCase(value: string): string {
-    resultMessage = `${CLI_STATE.Error} Use letters, numbers and '-' for project names (e.g. my-project-name)`;
-  }
-
-  return isValidProjectName ? true : resultMessage;
-}
-
-/**
- * Description: determine if string is valid component name
- * @param {string} value - a string value
- * @returns {any} -
- */
-function validateComponentName(value: any) {
-  const isString = typeof value === 'string';
-  const isNull = value === null || value.length === 0;
-  // characters in value are limited to alphanumeric characters and hyphens or underscores
-  const charactersMatch = value.match(/^[a-zA-Z0-9.\-_]+$/i) !== null;
-  const isValidComponentName = isString && charactersMatch;
-  let resultMessage;
-
-  if (isNull) {
-    resultMessage = `${CLI_STATE.Error} A component name is required`;
-  } else if (!charactersMatch) {
-    resultMessage = `${CLI_STATE.Error} Use letters, numbers and '-' for component names (e.g. my-component-name)`;
-  }
-
-  return isValidComponentName ? true : resultMessage;
-}
-
-/**
- * Description: determine if string is valid version name
- * @param {string} value - a string value
- * @returns {any} -
+ * @returns {string} - a string with the prefix '[rdvue]' removed
  */
-function validateVersionName(value: any) {
-  const isString = typeof value === 'string';
-  const isNull = value === null || value.length === 0;
-  // characters in value are limited to alphanumeric characters and hyphens or underscores
-  const charactersMatch = value.match(/^[a-zA-Z0-9.\-_]+$/i) !== null;
-  const isValidVersionName = isString && charactersMatch;
-  let resultMessage;
-
-  if (isNull) {
-    resultMessage = `${CLI_STATE.Error} A version name is required`;
-  } else if (!charactersMatch) {
-    resultMessage = `${CLI_STATE.Error} Use letters, numbers and '-' for version names (e.g. my-version-name)`;
-  }
-
-  return isValidVersionName ? true : resultMessage;
-}
-
-/**
- * Description: determine if string is valid page name
- * @param {string} value - a string value
- * @returns {any} -
- */
-function validatePageName(value: any) {
-  const isString = typeof value === 'string';
-  const isNull = value === null || value.length === 0;
-  // characters in value are limited to alphanumeric characters and hyphens or underscores
-  const charactersMatch = value.match(/^[a-zA-Z0-9.\-_]+$/i) !== null;
-  const isValidArgName = isString && charactersMatch;
-  let resultMessage;
-
-  if (isNull) {
-    resultMessage = `${CLI_STATE.Error} A page name is required`;
-  } else if (!charactersMatch) {
-    resultMessage = `${CLI_STATE.Error} Use letters, numbers and '-' for page names (e.g. page-name)`;
-  }
-
-  return isValidArgName ? true : resultMessage;
+function stripRdvuePrefix(value: string): string {
+  return value.replace('[rdvue]', '');
 }
 
 /**
- * Description: determine if string is valid service name
- * @param {string} value - a string value
- * @returns {any} -
+ * Description: Throwsan error with the provided message
```

Comment:
throws an*

Reply:
Thanks for the catch! The fix has been pushed